README.md

```diff
@@ -1,2 +1,8 @@
 # java-fundamentals
 labs for Java 401
+
+1. Basics
```

Comment:
Thanks for this listing! Note that if you look at the rendered readme file, the three lines labeled a/b/c all show up on the same line. Markdown, like HTML, will automatically collapse text all into the same line. If you want these to show on separate lines, you can use bullet points (either `-` or `*`) to make a UL, which will keep things on separate lines.